packages/react/src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -92,11 +92,16 @@ const iconStyles: ComponentSlotStylesInput<IconProps, IconVariables> = {
       backgroundColor: v.backgroundColor,
       boxSizing: isFontBased ? 'content-box' : 'border-box',
 
+      // overriding the base theme default rtl behavior as in teams theme the svg/svgFlippingInRtl slots are used for this
+      ...((rtl || !isFontBased) && {
```

Comment:
the comment and the expression itself mean different things - the comment suggests that these styles should go in RTL only, but the condition used doesn't guarantee that (if icon is not a font-based one, the styles will be applied both in RTL and LTR). Lets sync the comment and the condition - probably, the intent was to do just the following:

```js
...(!isFontBased) { .. }

```